editor.planx.uk/src/@planx/components/PlanningConstraints/Public.tsx

```diff
@@ -284,19 +290,18 @@ export function PlanningConstraintsContent(
             >
               <ConstraintsList data={negativeConstraints} metadata={metadata} />
             </SimpleExpand>
-            <PlanningConditionsInfo />
+            <Disclaimer text={disclaimer} />
           </>
         )}
     </Card>
   );
 }
 
-const PlanningConditionsInfo = () => (
+const Disclaimer = (props: { text: string }) => (
   <WarningContainer>
     <ErrorOutline />
-    <Typography variant="body1" ml={2} fontWeight={FONT_WEIGHT_SEMI_BOLD}>
-      This page does not include information about historic planning conditions
-      that may apply to this property.
+    <Typography variant="body1" ml={2}>
+      {props.text || DEFAULT_PLANNING_CONDITIONS_DISCLAIMER}
```

Comment:
I've introduced `disclaimer` as a required prop, therefore in order for currently published services to not have a visual regression it's necessary to pass the `DEFAULT_` text here as a fallback. 

I've also removed the `fontWeight` since it's a rich text input!